src/frontend/scss/objects/_editor.scss

```diff
@@ -246,3 +246,16 @@ addImage Editor
     transform: scale(1);
   }
 }
+.latex-inline,
+.latex-inline:hover {
```

Comment:
Do you need the `:hover` selector to overwrite hover behavior ?
From user point of view, this is weird to remove hover effect. Here I would have rather add an effect on hover. (e.g: modify opacity ?)

Reply:
Yes, I add to overwrite because of draft.js otherwise I had a gray square instead. True, and good idea I can still implement an opacity property on the over ver